packages/scaffold-ui/src/partials/w3m-account-wallet-features-widget/index.ts

```diff
@@ -265,10 +241,6 @@ export class W3mAccountWalletFeaturesWidget extends LitElement {
     })
     RouterController.push('WalletSend')
   }
-
```

Comment:
given we are removing access to the route, we shuold remove all unnecessary views and specific components to reduce bundle